nuxt.config.js

```diff
@@ -138,13 +138,26 @@ export default {
   ],
 
   // Modules (https://go.nuxtjs.dev/config-modules)
-  modules: ["@nuxtjs/google-fonts"],
+  // socail-sharing (https://www.npmjs.com/package/vue-social-sharing)
+  modules: ["@nuxtjs/google-fonts", "vue-social-sharing/nuxt"],
```

Comment:
Shouldn’t need to do this, this should just be for nuxt modules.

Reply:
im like 90% i put that in there because the console was giving me an error and yelling at me. Or maybe it was just vscode being goofy. will take it out and see what happens.